MediaBrowser.Controller/MediaEncoding/EncodingHelper.cs

```diff
@@ -276,6 +276,21 @@ private bool IsVideoToolboxFullSupported()
                 && _mediaEncoder.SupportsFilter("scale_vt");
         }
 
+        private bool IsSwTonemapAvailable(EncodingJobInfo state, EncodingOptions options)
+        {
+            if (state.VideoStream is null
+                || !options.EnableTonemapping
+                || GetVideoColorBitDepth(state) != 10
+                || !_mediaEncoder.SupportsFilter("tonemapx")
+                || !(string.Equals(state.VideoStream?.ColorTransfer, "smpte2084", StringComparison.OrdinalIgnoreCase) || string.Equals(state.VideoStream?.ColorTransfer, "arib-std-b67", StringComparison.OrdinalIgnoreCase)))
```

Comment:
Are these checks duplicated?
```
|| !(string.Equals(state.VideoStream?.ColorTransfer, "smpte2084", StringComparison.OrdinalIgnoreCase) || string.Equals(state.VideoStream?.ColorTransfer, "arib-std-b67", StringComparison.OrdinalIgnoreCase)))
```
```
            return state.VideoStream.VideoRange == VideoRange.HDR
                   && state.VideoStream.VideoRangeType is VideoRangeType.HDR10 or VideoRangeType.HLG or VideoRangeType.DOVIWithHDR10 or VideoRangeType.DOVIWithHLG;
```

Reply:
The former checks for all supported HDR curves and the latter filters out Dolby vision videos without fallback profiles. Only checks for the latter should be enough but we are doing this double check for hardware tone mapping as well.

Reply:
https://github.com/jellyfin/jellyfin/blob/eb45a36af0ff3ef540a4a936ea1329139f0523c1/MediaBrowser.Model/Entities/MediaStream.cs#L703-L755

`ColorTransfer` is normalized to `VideoRange` and `VideoRangeType`. We should eventually avoid using `ColorTransfer` directly.